library/src/cache/patch_manager.rs

```diff
@@ -316,19 +314,6 @@ impl PatchManager {
 }
 
 impl ManagePatches for PatchManager {
```

Comment:
If you're making this a reading/writing class (or intend it to be), I would leave yourself a comment to explain that goal.